src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+}
+
+body {
+  margin: 0;
+  font-family: "Avenir", sans-serif;
+}
+
+.bigSearch {
+  margin: 20px 72px 20px 8px;
+}
+
+.smallSearch {
+  margin: 20px 43px 10px 8px;
+}
+
+input {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+}
+
+.input--big {
+  height: 66px;
+  padding-left: 62px;
+}
+
+.input--small {
+  height: 38px;
+  padding-left: 33px;
+}
+
+input:hover {
+  border-color: rgb(142, 64, 64);
+}
+
+input:focus {
+  border-color: rgb(23, 230, 23);
+  font-family: "Avenir", sans-serif;

```

Comment:
You should inherit the font, there is no need to repeat it all over the code

Reply:
I changed styles as in figma, without font.